source/except.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+Create a type from an object type without certain keys.
+
```

Comment:
Same as https://github.com/sindresorhus/type-fest/pull/44/files#r300570410 here, but also include how it differs.

Reply:
Also a note to encourage people to vote and comment on https://github.com/microsoft/TypeScript/issues/30825 if they want this as a built-in helper.